Makefile

```diff
@@ -123,4 +132,14 @@ cluster/prepare/local: cluster/prepare
 	-oc create -f deploy/crds/*.crd.yaml
 	@oc create -f deploy/service_account.yaml
 	@oc create -f deploy/role.yaml
-	@oc create -f deploy/role_binding.yaml
\ No newline at end of file
+	@oc create -f deploy/role_binding.yaml
+
+.PHONY: cluster/prepare/olm
+cluster/prepare/olm: cluster/prepare
+	oc process -p NAMESPACE=$(NAMESPACE) -f deploy/operator-source-template.yml | oc create -f - -n openshift-marketplace
+	oc process -p NAMESPACE=$(NAMESPACE) -f deploy/operator-subscription-template.yml | oc create -f - -n $(NAMESPACE)
```

Comment:
I had to split the template up into 2 as the operator source and catalog config were being created in the wrong namespace, apparently namespaces are removed from templates during processing and you cant have it create things in more than one ns :man_shrugging: 

Namespace creation is also not in these templates as that was causing issues, so it's being created ahead of time.